client/tests/integration/triggers/time_trigger.rs

```diff
@@ -170,6 +170,7 @@ fn pre_commit_trigger_should_be_executed() -> Result<()> {
 }
 
 #[test]
+#[ignore = "Only on nightly"]
```

Comment:
```suggestion
#[ignore = "nightly"]
```
what is the meaning of the string attached?

Reply:
If you run `cargo test` on 1.62+ (nightly) you'll get the string `ignored, nightly`, so people running the test know why it's ignored.